quick_start.rst

```diff
@@ -412,6 +399,9 @@ If you executed ``--append-snippets``, your ``FeatureContext`` should look like:
     and ``I should have :arg1 products in the basket`` into one
     ``I should have :arg1 product(s) in the basket``.
 
+    Also, Behat will not add the ``use`` statements for the step attributes for you, you
+    may need to add them manually if they were not yet present in your context.
```

Comment:
How hard would it be to make it that it does? If that's too hard, should we generate attributes with FQCN and it's up to the user to shorten them if they want?

I can see it tripping a lot of people up if their file looks right but it's not matching anything because they've not noticed they need to import the attributes...

Reply:
I have added an issue to our Behat repo to handle this. Until this issue is done we can keep this comment here. The issue is: https://github.com/Behat/Behat/issues/1553

Reply:
I think this is now going to be solved in Behat/Behat#1549 so we can probably take it out?